clients/admin-ui/src/features/system/privacy-declarations/PrivacyDeclarationForm.tsx

```diff
@@ -264,6 +329,7 @@ export const PrivacyDeclarationForm = ({
       onSubmit,
       initialValues: passedInInitialValues,
       allDataUses: dataProps.allDataUses,
+      privacyDeclarationId: passedInInitialValues?.id,
```

Comment:
it seems like the hook should be able to grab the id based on `passedInInitialValues` since it's already getting that as a child (rather than having to pass in another prop with the id separately